build/media_source/plg_system_guidedtours/js/guidedtours.es6.js

```diff
@@ -0,0 +1,407 @@
+    showTourInfo(tour, obj.steps[0]);
+    ind = 1;
+  }
+
+  // Now let's add all follow up steps
+  const len = obj.steps.length;
+  let buttons;
+
+  // eslint-disable-next-line no-plusplus
+  for (let index = ind; index < len; index++) {
+    buttons = [];
+
+    // If we have at least done one step, let's allow a back step
+    // - if after the start step
+    // - if not the first step after a form redirect
+    // - if after a simple redirect
+    if (prevStep === null || index > ind || obj.steps[index].type === 'redirect') {
+      addBackButton(buttons, obj.steps[index]);
+    }
+
+    if (
+      obj
+      && obj.steps[index].target
+      && obj.steps[index].type === 'interactive'
+    ) {
+      const ele = document.querySelector(obj.steps[index].target);
+      if (ele) {
+        if (obj && obj.steps && obj.steps[index] && obj.steps[index].interactive_type) {
+          switch (obj.steps[index].interactive_type) {
+            case 'submit':
+              ele.addEventListener('click', () => {
+                sessionStorage.setItem('currentStepId', obj.steps[index].id + 1);
+              });
+              break;
+
+            case 'text':
+              ele.step_id = index;
+              ele.addEventListener('input', (event) => {
+                if (event.target.value.trim().length) {
+                  enableButton(event);
+                } else {
+                  disableButton(event);
+                }
+              });
+              break;
+
+            case 'button':
+              tour.next();
+              break;
+
+            case 'other':
+            default:
+              break;
+          }
+        }
+      }
+    }
+
+    if (index < len - 1) {
+      let disabled = false;
+      if (obj && obj.steps[index].interactive_type === 'text') {
+        disabled = true;
+      }
+      if (
+        (obj && obj.steps[index].type !== 'interactive')
+        || (obj && obj.steps[index].interactive_type === 'text')
+        || (obj && obj.steps[index].interactive_type === 'other')
+      ) {
+        pushNextButton(buttons, obj.steps[index], disabled, disabled ? 'disabled' : '');
+      }
+    } else {
+      pushCompleteButton(buttons);
+    }
+
+    addStepToTourButton(tour, obj.steps[index], buttons);
+    prevStep = obj.steps[index];
+  }
+
+  tour.start();
+}
+
+function loadTour(tourId) {
+  if (tourId > 0) {
+    const url = `${Joomla.getOptions('system.paths').rootFull}administrator/index.php?option=com_ajax&plugin=guidedtours&group=system&format=json&id=${tourId}`;
+    fetch(url)
+      .then((response) => response.json())
+      .then((result) => {
+        if (!result.success) {
+          if (result.messages) {
+            Joomla.renderMessages(result.messages);
+          }
+
+          // Kill all tours if we can't find it
+          emptyStorage();
+        }
+        startTour(result.data);
+      })
+      // eslint-disable-next-line no-console
+      .catch((error) => console.error(error));
```

Comment:
```suggestion
      .catch((error) => {
        throw new Error(error)
      });
```

Also remove the eslint disable above

Reply:
Thanks!